languages/csharp/exercises/concept/properties/.meta/design.md

```diff
@@ -0,0 +1,193 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the student to C# properties and to teach the student how the concept of properties is implemented in [C#][docs.microsoft.com-properties]. We'll teach the student about properties by having the student work with classes that expose contrasting forms of properties. The students will learn to work with properties with backing fields and auto-implemented properties.
+
+Properties are covered early in the C# track as their purpose and power can be shown with few dependencies (classes, access modifiers and fields of simple types).
+
+
+## Things to teach
+
+- public properties
+- properties with backing fields
+- auto-implemented properties 
+- input verification
+- calculated properties
+- get-only properties vs. properties with private set accessors
+- set-only properties vs. properties with private get accessors
+- properties exposing reference types
```

Comment:
So this list looks good, but I'd rephrase it a bit differently to correspond to how we [phrase things in other exercises](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/numbers/.meta/design.md#learning-objectives):

So my list would look similar to your list, but slightly differently:

```suggestion
- Know what properties are and how they related to fields and methods.
- Know what backing-field properties are.
- Know what auto-implemented properties are.
- Know what calculated properties are.
- Know how to use property accessors to customize visibility.
- Know how to define the different types of properties.
```

I feel that the above list focuses more on the concepts involved than the implementation. Does that make sense?